forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -224,6 +230,11 @@ open class DeviceBindingCallback : AbstractCallback, Binding {
             return
         }
 
+        if (!deviceAuthenticator.validateCustomClaims(customClaims)) {
+            handleException(DeviceBindingException(Unsupported("Invalid custom claims")))
```

Comment:
For iOS we consider this as Abort right? Should align with iOS.

Reply:
I changed to align this with iOS. Added a new InvalidCustomClaims Exception type with Abort client error